h2o-py/h2o/utils/progressbar.py

```diff
@@ -122,7 +123,7 @@ def __init__(self, title=None, widgets=None, maxval=1.0, file_mode=None, hidden=
         self._next_poll_time = None
 
 
-    def execute(self, progress_fn):
+    def execute(self, progress_fn, verbose_model=None):
```

Comment:
I think a little nicer would be to pass a lambda function that would get the model info and print the information. This way we would better separate concerns - progress bar wouldn't need to know anything about h2o.

I suggest to change the function this way:

```
def execute(self, progress_fn, print_verbose_info=None):
  ...
  if print_verbose_info is not None:
     print_verbose_info()
```

The imports of this class would remain the same as before.